### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Fixes some styling to match the intended design.

- Overflowing image
- SVG logo
- Better line breaks in homepage copy with non-breaking spaces `&nbsp;`

## How can we measure success?

- Better DevX &rarr; Much more thorough use of `variables.ts`
- Minimal visual changes &rarr; assets optimisation.

## Images

| before    | after    |
|-----------|----------|
|![before][]|![after][]| 

[before]: https://user-images.githubusercontent.com/76776/95569072-f9214000-0a1c-11eb-8344-b2ac4b1b70fc.png
[after]: https://user-images.githubusercontent.com/76776/95566606-b27e1680-0a19-11eb-909e-3f920ac5168b.png

### CODE REVIEW

File: src/styles/variables.ts [hunk 1]
```diff
@@ -30,7 +30,8 @@ export const fonts = {
   sansSerif:
     '-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif',
   serif: 'Georgia, "Times New Roman", Times, serif',
-  monospace: 'Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace, monospace'
+  monospace: 'Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace, monospace',
+  headline: 'GH Guardian Headline, Georgia, "Times New Roman", Times, serif'
```
Comment (buck06191) [inline src/styles/variables.ts:34]: Can we get the Guardian font from `source`?

**Review by buck06191 — APPROVED** [review]
LGTM 
Just need to look at the Guardian fonts

File: src/styles/variables.ts (hunk 1 above)
Comment (lucymonie) [reply src/styles/variables.ts:34]: https://theguardian.design/2a1e5182b/p/930d69-overview

Comment (buck06191) [reply src/styles/variables.ts:34]: Thanks @lucymonie 

@mxdvl Maybe this will help: https://theguardian.design/2a1e5182b/p/075107-foundations/t/89a552

**Comment by buck06191** [comment]
I'm going to merge this and then pick up the fonts work separately

**Pull request merged**